rclpy/rclpy/impl/implementation_singleton.py

```diff
@@ -28,5 +28,17 @@
 
 import importlib
 
-
-rclpy_implementation = importlib.import_module('._rclpy', package='rclpy')
+module_name = '._rclpy'
+
+try:
+    rclpy_implementation = importlib.import_module(module_name, package='rclpy')
+except ImportError as e:
+    import os
+    if os.path.isfile(e.path):
+        import sys
+        print(
+            "The module '%s' failed to be imported while being present on the system."
+            " Please refer to '%s' for possible solutions\n" %
+            (module_name, 'https://github.com/ros2/ros2/wiki/Rclpy-Import-error-hint'),
```

Comment:
I would rather print `e.path` here since `module_name` is already part of the exception string. The message could start with "The C extension '%s' failed ..." then.